framework/lib/components/toolbox/toolbox.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { useMultiStyleConfig, useToken } from '@chakra-ui/system'
 import { coreZIndex } from '@northlight/tokens'
 import { FocusScope } from '@react-aria/focus'
+import { VStack } from '@chakra-ui/react'
```

Comment:
![Screenshot 2024-08-02 at 11 14 45](https://github.com/user-attachments/assets/679d3413-292e-41b2-b70b-3247ae5e8cc1)
it should be imported from lib/stack